base/cortex-common/enginei.h

```diff
@@ -36,7 +36,8 @@ class EngineI {
     }
     return false;
   }
-  virtual void SetFileLogger() = 0;
+  virtual void SetFileLogger(int max_log_lines,
```

Comment:
Please move this function to the end of the class

Reply:
I'll remove it